client/consensus/aura/src/lib.rs

```diff
@@ -143,6 +143,7 @@ where
 				telemetry,
 				block_proposal_slot_portion,
 				max_block_proposal_slot_portion,
+				compatibility_mode: sc_consensus_aura::CompatibilityMode::None,
```

Comment:
```suggestion
				compatibility_mode: CompatibilityMode::None,
```
Silly nit :-) (feel free to ignore). Just to be inline with the way the symbol is imported in the import queue source.
Obviously this requires to adjust the `sc_consensus_aura` import list at the begin of the file